README.md

```diff
@@ -1,9 +1,159 @@
+
+We now have access to all of the gems that we need, but we still need to setup a connection to our database. Add the following block of code to your `environment.rb` file. 
+
+```ruby
+configure :development do
+  set :database, "sqlite3:db/database.db"
+end
+```
+
+This sets up a connection to a sqlite3 database named "database.db", located in a folder called "db." If we wanted our `.db` file to be called `dogs.db`, we could simply change the name of this file:
+
+```ruby
+configure :development do
+  set :database, "sqlite3:db/dog.db"
+end
+```
+
+But for now, `database.db` is a great name! Notice that this didn't actually create those files or folders yet - that's what Rake will help us with!
+
+### Making a Rakefile
+
+As we mentioned, `rake` gives us the ability to quickly make files and setup automated tasks. We define these in a file called `Rakefile`. First, create a `Rakefile` in the root of our project directory. In your `Rakefile`, we'll require our `config/environment.rb` file to load up our environment, as well as `"sinatra/activerecord/rake"` to get Rake tasks from the `sinatra-activerecord` gem.
+
+```ruby
+require'./config/environment'
+require "sinatra/activerecord/rake"
+```
+
+In the terminal, type `rake -T` to view all of your available rake tasks. You should see the following output:
+
+```bash
+rake db:create              # Creates the database from DATABASE_URL or config/database.yml for...
+rake db:create_migration    # Create a migration (parameters: NAME, VERSION)
+rake db:drop                # Drops the database from DATABASE_URL or config/database.yml for t...
+rake db:fixtures:load       # Load fixtures into the current environment's database
+rake db:migrate             # Migrate the database (options: VERSION=x, VERBOSE=false, SCOPE=blog)
+rake db:migrate:status      # Display status of migrations
+rake db:rollback            # Rolls the schema back to the previous version (specify steps w/ S...
+rake db:schema:cache:clear  # Clear a db/schema_cache.dump file
+rake db:schema:cache:dump   # Create a db/schema_cache.dump file
+rake db:schema:dump         # Create a db/schema.rb file that is portable against any DB suppor...
+rake db:schema:load         # Load a schema.rb file into the database
+rake db:seed                # Load the seed data from db/seeds.rb
+rake db:setup               # Create the database, load the schema, and initialize with the see...
+rake db:structure:dump      # Dump the database structure to db/structure.sql
+rake db:structure:load      # Recreate the databases from the structure.sql file
+rake db:version             # Retrieves the current schema version number
+```
+Awesome!
+
+
+### Testing it Out
+
+Let's test out our handiwork by creating a `dogs` table with two columns: `name` and `breed`. First, let's create our migration:
+
+```bash
+rake db:create_migration NAME=create_dogs
+```
+You should see the following output:
+
+```bash
+=># db/migrate/20150914201353_create_dogs.rb
```

Comment:
should we clarify that the number in front of the migration file will be different than what is shown here?